spec/patterns/core_spec.rb

```diff
@@ -234,6 +234,11 @@
       expect( event.to_hash['tags'] ).to include '_grokparsefailure'
     end
   end
+
+  it "matches paths with non-ascii characters" do
+    expect(grok_match(pattern, '/opt/Čierný_Peter/.中', true)).to pass
```

Comment:
I'm not terribly confident in the `to pass` bit, especially as our `pattern` appears to be unanchored. Can we favor using named captures and `to include(`...`)` instead to set firmer expectations about exactly what is matched/captured?

Reply:
actually is anchored, the last `exact_match = true` [causes anchoring](https://github.com/logstash-plugins/logstash-patterns-core/blob/0204239/spec/spec_helper.rb#L36).

agree it's probably not super clear but this is what we've started with ...